### DISCUSSION TITLE
How can I show the menu

### DISCUSSION THREAD

**Discussion by whereissam** [nunocoracao/blowfish#1013] [Q&A]
I follow the steps in https://blowfish.page/docs/configuration/ than go to menus.en.toml to set the menu. When I uncomment menu options but show nothing in my screen

`
[[main]]
 name = "Blog"
 pageRef = "posts"
 weight = 10
 `
 
 Does it only need to be change in themes/blowfish/config/_default/file ? 
 
 For instance, I follow the step to change my title and image which is position in the middle. I can't see anything until I copy the setting to my root hugo.toml than I can see the change. I am a little confuse, please if someone know about that can help me. Thanks!
 

![image](https://github.com/nunocoracao/blowfish/assets/63341022/3a4fd1f3-93d8-49aa-a73a-19b50cde1451)

**Comment by JahsehJ** [comment]
Hello! You should not modify any of the blowfish theme files, copy the configuration files in `themes/blowfish/config/_default/` to `config/` instead. Hope this helps, feel free to leave more comments.

**Comment by nunocoracao** [comment]
@whereissam  please have a look at the examples for the Blowfish site and website. I think the original configuration had something wrong. Looking at an example will help

**Reply by JahsehJ** [reply]
Sorry but you replied to the wrong person 😂

**Reply by nunocoracao** [reply]
Thanks @JahsehJ fixed it - my bad

**Reply by nunocoracao** [reply]
Additionally, @whereissam I wrote a step-by-step guide for new users
here - https://n9o.xyz/posts/202310-blowfish-tutorial/
